subprojects/core/src/main/java/org/gradle/api/internal/initialization/transform/BaseInstrumentingArtifactTransform.java

```diff
@@ -78,16 +78,23 @@ public void transform(TransformOutputs outputs) {
             return;
         }
 
+        InjectedInstrumentationServices injectedServices = getObjects().newInstance(InjectedInstrumentationServices.class);
+        if (getParameters().getAgentSupported().get()) {
```

Comment:
💅 can we extract `getParameters().getAgentSupported().get()` into a method?

Reply:
Makes sense, fixed